src/enclave/enclave_oe.c

```diff
@@ -295,7 +295,7 @@ void sgxlkl_ethread_init(void)
     _lthread_sched_init(sgxlkl_enclave_state.config->stacksize);
     lthread_run();
 
-    return;
+    return sgxlkl_enclave_state.exit_status;
```

Comment:
Please add a comment explaining this.  I guess it's so that one ethread exits while another can be stuck in a non-yielding loop, and the first one to exit provides the return value?

Reply:
Yes, we don't know which of our ethreads (either the first `einit` ethread or any of the subsequent ethreads) will be exiting, so we need to propagate the exit code on both code paths.